modules/ding_event/ding_event.admin.inc

```diff
@@ -6,27 +6,44 @@
 
 /**
  * Get a form to select currency (Euro | Kr)
- * @param $form
- * @param $form_state
+ *
+ * @param array $form
+ *   An associative array containing the structure of the form.
+ * @param array $form_state
+ *   A keyed array containing the current state of the form.
  * @return mixed
  */
-function ding_event_admin_currency_setting_form($form, &$form_state) {
+function ding_event_admin_setting_form(array $form, array &$form_state) {
   $currency = variable_get('ding_event_currency_type', 'Kr');
 
-  $form['ding_event_currency'] = array(
+  $form['ding_event'] = array(
     '#type' => 'fieldset',
     '#title' => t('Event currency'),
```

Comment:
This fieldset currently spans the entire form (currency + frontpage items) but the title no longer fits all the child elements.

I suggest one of two options:

1. Drop the fieldset
2. Add another fieldset for frontpage items